nathan-shulkin.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html lang="en">
+<link rel="stylesheet" href="nathan-shulkin.css">
+  <head>
+    <title>CSS Timeline</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <h1>Brief History of CSS</h1>
+    <ul>
+      <li><div id="green"><div class="chrono">1994</div>: The first work on style sheets started at CERN.</div></li>
+      <li><div class="chrono">December 1996</div>: CSS was proposed as W3C standard:
+        <blockquote>
+          "CSS1 defines the language in which style sheets are written and a set of properties that designers can use to describe how documents should be presented. The properties in CSS1 cover the most common screen-based presentations."
+        </blockquote>
+      </li>
+      <li><div class="chrono">November 1997</div>: The W3C released the first public CSS2 Working Draft.</li>
```

Comment:
For the purposes of styling parts of inline content, we use the `span` element, rather than `div`s.

Here's more info on using `span` elements: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span